### DISCUSSION TITLE
Keep new notification buttons in the notification history

### DISCUSSION THREAD

**Discussion by albakhtari** [cstayyab/WALC#200] [Feature Requests]
Like how the browsers do it:

![image](https://user-images.githubusercontent.com/79008923/172995678-d8a524ff-ad2d-4f5f-b525-2c42cd239cd8.png)

On WALC, the buttons are only shown when the notification is in it's floating state....

**Comment by daffamumtaz2361** [comment]
https://bugs.kde.org/show_bug.cgi?id=407361

It seems WALC need to send `resident` hint in the dbus call
